client/src/SubHeader.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
```

Comment:
so yeah, i didn't really write any of the code from this file, mostly just copied from where we have other sub headers (usually with a 'back' and search input)
however we have inconsistent sub headers so will be bringing this to the design team to hopefully clean up
will reuse this component in all the other places in a separate PR